board/miyoo/scripts/genimage.sh

```diff
@@ -60,6 +60,9 @@ export IMAGE_NAME="${BR2_VENDOR}-${CFW_TYPE}-${CFW_RELEASE}-${CFW_HASH}_${LIBC}-
 # Relocate board files for genimage-sdcard config to read (see last cmd)
 cp -r board/miyoo/boot "${BINARIES_DIR}"
 cp -r board/miyoo/main "${BINARIES_DIR}"
+if [ "${LIBC}" = "uClibc-ng" ]; then
+    rm -rf "${BINARIES_DIR}/main/apps/gstreamer"
```

Comment:
another reason to ditch gst-raw.py in favor of simply gst-play, or you could simply add this script in package/gstreamer1 as a patch and use TARGET_INSTALL